WeakAuras/AuraEnvironment.lua

```diff
@@ -435,7 +758,7 @@ local exec_env = setmetatable({},
     elseif overridden[k] then
       return overridden[k]
     else
-      return _G[k]
+      return _G[k] -- proxifier(_G[k])
```

Comment:
Right now this proxifier does nothing.

Reply:
Because it is not needed. Virtual regions is a thing